### ISSUE TITLE
Setting flankVersion(String) doesn't seem to be effective

### ISSUE THREAD

**Issue by jasonschroeder-sfdc** [runningcode/fladle#56]
My Fladle extension config:

```groovy
fladle {
    ....
    flankVersion("6.2.0")
    ....
}
```

Then, I run my tests:

```sh
./gradlew runFlank
```

**Expectation:** Gradle Build Scan shows a dependency on `flank:flank:6.2.0`. Flank v6.2.0 is invoked 
**Actual results:** Gradle Build Scan shows a dependency on `flank:flank:5.1.0`, which appears to be the default value.

I've also tried `v6.2.0` and `banana`, neither of which provided any different results. Flank 5.1.0 is always used.

My environment:
Fladle 0.6.2, Gradle 5.4.1, macOS Mojave 10.14.5

**Comment by jasonschroeder-sfdc** [comment]
Thank you!

**Comment by runningcode** [comment]
Thanks for the detailed bug report! This has been fixed in 0.6.3, let me know if it works.
